google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -2501,6 +2579,46 @@ private void removeLongRunningSessions(
         }
       }
     }
+
+    void maintainMultiplexedSession(Instant currentTime) {
+      try {
+        if (options.getUseMultiplexedSession()) {
+          synchronized (lock) {
+            if (multiplexedSession.get() != null && isMultiplexedSessionStale(currentTime)) {
+              logger.log(
+                  Level.INFO,
+                  String.format(
+                      "Removed Multiplexed Session => %s created at => %s",
+                      multiplexedSession.get().getName(),
+                      multiplexedSession.get().getDelegate().getCreateTime()));
+              if (multiplexedSessionRemovedListener != null) {
+                multiplexedSessionRemovedListener.apply(multiplexedSession.get());
+              }
+              /*
+               This will attempt to create a new multiplexed session. if successfully created then
+               the existing session will be replaced. Note that there maybe active transactions
+               running on the stale session. Hence, it is important that we only replace the reference
+               and not invoke a DeleteSession RPC.
+              */
+              maybeCreateMultiplexedSession();
+            }
+          }
+        }
+      } catch (final Throwable t) {
+        logger.log(Level.WARNING, "Failed to maintain multiplexed session", t);
+      }
+    }
+
+    boolean isMultiplexedSessionStale(Instant currentTime) {
+      final MultiplexedSession session = multiplexedSession.get();
+      final Duration durationFromCreationTime =
+          Duration.between(session.getDelegate().getCreateTime(), currentTime);
+      if (durationFromCreationTime.compareTo(options.getMultiplexedSessionMaintenanceDuration())
+          > 0) {
+        return true;
+      }
+      return false;
```

Comment:
nit: just return the value from the comparison, instead of using an if statement

Reply:
Done.